spec.bs

```diff
@@ -1038,6 +1085,7 @@ partial interface SharedStorageWorkletGlobalScope {
 dictionary SharedStoragePrivateAggregationConfig {
   USVString aggregationCoordinatorOrigin;
   USVString contextId;
+  [EnforceRange] unsigned long long filteringIdMaxBytes;
```

Comment:
Is there a reason to use long long for max bytes?

Reply:
That's a good point, doesn't JS max out at 54 bits for ints because that's the non-exponent part of a double (and all JS numbers are doubles)? Is an unsigned long enough?

Reply:
This is just following the guidance in the Web Platform Design Principles [here](https://www.w3.org/TR/design-principles/#numeric-types). We unfortunately didn't follow this for some of the other existing fields -- we may want to change those at some point.